VideoApi/VideoApi/Controllers/ConferenceManagementController.cs

```diff
@@ -187,5 +187,10 @@ public async Task<IActionResult> TransferParticipantAsync(Guid conferenceId, Tra
                 return StatusCode(ex.StatusCode, ex.Response);
             }
         }
+
+        private string TransferFromRoomType(ParticipantBase participant)
+        {
+            return (participant.UserRole == UserRole.QuickLinkParticipant || participant.UserRole == UserRole.QuickLinkObserver) && participant.CurrentConsultationRoom != null ? participant.CurrentConsultationRoom.Label : RoomType.WaitingRoom.ToString();
```

Comment:
I wonder if we could just have the second part of this check? I get that we only care about quick link users in this ticket.

I'll leave that up to you. If we keep this we should test when it is a quick link user and the consultation room is null that we get the waiting room string